Update goals controller to display view for user goals

MichaelRezene opened the pull request:
- [x] Update goals controller to display view for user goals
- [x] create the view form
- [x] styled /register
- [x] styled /login
- [x] Modified /goals/create to /goals/set
- [x] styled /goals/set
- [x] Created nav bar to /login
- [x] Created nav bar to /register
- [x] Created nav bar to /goals/set

MichaelRezene committed:
Update goals controller to display view for user goals

MichaelRezene committed:
THP Update Goals controller to redirect to the create goals if no goals present

MichaelRezene committed:
- Created the home controller class

MichaelRezene committed:
- Created view goals form and tested
- created css package and css file (site.css)

MichaelRezene committed:
- Created view goals form and tested
- created css package and css file (site.css)

MichaelRezene committed:
Merge origin/partials-create branch

MichaelRezene committed:
- Styled registration form
- styled view form
- styled create form

MichaelRezene committed:
- Styled /registration
- Modified /goals/create to /goals/set
- styled /login
- styled /goals/set

MichaelRezene committed:
- Styled /registration
- Modified /goals/create to /goals/set
- styled /login
- styled /goals/set

ItsRileyB reviewed (changes requested):
Looking good, work on the changes we discussed and we'll take a look.

MichaelRezene committed:
- Created nav bar to /login
- Created nav bar to /register
- Created nav bar to /goals/set
- styled buttons

MichaelRezene committed:
- Created nav bar to /login
- Created nav bar to /register
- Created nav bar to /goals/set
- styled buttons

MichaelRezene committed:
merge master branch to update branch with newest changes

MichaelRezene committed:
- Modified set goals
- connect /login to /partials/navbar
connect /register to /partials/navbar
connect /goals/set to /partials/navbar

ItsRileyB committed:
Merge branch 'master' into create-goals

MichaelRezene committed:
- Modified set goals
- connect /login to /partials/navbar
connect /register to /partials/navbar
connect /goals/set to /partials/navbar

MichaelRezene committed:
Merge branch 'create-goals' of github.com:MPTDesigns/nutritrackr into create-goals

ItsRileyB reviewed (approved):
Everything looks good, Mike! I'll merge the changes now.
